src/Files.App/Utils/RecentItem/RecentItems.cs

```diff
@@ -114,7 +113,6 @@ public async Task<List<RecentItem>> ListRecentFilesAsync()
 		/// </summary>
 		public async Task<List<RecentItem>> ListRecentFoldersAsync()
 		{
-			var recentItems = new List<RecentItem>();
```

Comment:
Is this related, or should it be reverted?

Reply:
The variable was previously unused, so the change might as well stay